Packs/SumoLogic_Cloud_SIEM/ReleaseNotes/1_1_19.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Sumo Logic Cloud SIEM
+- Remove fields having empty list `[]`, empty string `""`, empty tuple `()`, or empty object `{}` as a value.
```

Comment:
Please indicate which command or flow the change affects. 
In addition, please add a line to the Docker image update.

Reply:
I have updated the release notes as per your comments. Please take a look.